styles/cobalt/catppuccin.user.css

```
@@ -0,0 +1,93 @@
+/* ==UserStyle==
+@name cobalt Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/cobalt
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/cobalt
+@version 1.0.0
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/cobalt/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Acobalt
+@description Soothing pastel theme for cobalt
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain('cobalt.tools') {
+
+  :root[data-theme="dark"] {
```

```suggestion
@-moz-document domain('cobalt.tools') {
  :root[data-theme="dark"] {
```

Sorry for the whitespace nit :)